target_chains/solana/program_simulator/src/lib.rs

```diff
@@ -0,0 +1,76 @@
+use {
+    solana_program::{
+        hash::Hash,
+        instruction::Instruction,
+        pubkey::Pubkey,
+        system_instruction,
+    },
+    solana_program_test::{
+        BanksClient,
+        BanksClientError,
+        ProgramTest,
+        ProgramTestBanksClientExt,
+    },
+    solana_sdk::{
+        signature::{
+            Keypair,
+            Signer,
+        },
+        transaction::Transaction,
+    },
+};
+
```

Comment:
I dropped the logic to add a program as upgradable because it doesn't seem to work with 1.16.20, and I don't think we'll need this for the Solana Receiver